package.json

```diff
@@ -5,10 +5,10 @@
     "bs-custom-file-input": "^1.3.4",
     "jquery-simulate": "^1.0.2",
     "js-cookie": "^3.0.0",
-    "leaflet": "^1.8.0",
+    "leaflet": "^1.9.4",
```

Comment:
Should we uncomment this code now that we’re OK with newer Leaflet versions?

https://github.com/OpenHistoricalMap/ohm-website/blob/a71a6b795dda0d9c908495cf8862fdb8e9bf02e3/Vendorfile#L6